src/Sulu/Bundle/MediaBundle/Resources/js/containers/MultiMediaDropzone/MultiMediaDropzone.js

```diff
@@ -80,12 +84,18 @@ class MultiMediaDropzone extends React.Component<Props> {
         });
 
         return Promise.all(uploadPromises).then((...media) => {
```

Comment:
This seems to behave a a bit strange, because this promise already seems to be resolved when the first request fails... I guess that could lead to some inconsistent state after reloading the view, because some uploads might still finish afterwards. Shouldn't we maybe use something like [`Promise.allSettled`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled) instead?

Reply:
yes, that would be better 🙂 
what should we pass to the `onUploadError` in that case? the first error or all errors?

Reply:
All errors would probably make more sense, wouldn't it? But I guess it doesn't make a huge difference anyway, since the handling will not have a look at every single error, right?

Reply:
no in this case it will not, but maybe we will reuse the component somewhere and i am not sure what i would expect then 🙂
another possibility would be calling the callback for each error that happens? 🤔

Reply:
Personally I would have expected all the errors at once... But that's subjective I guess. The advantage of that would be that it would be easier to handle in the calling component. If the callback is called every single time you would probably have to keep an array of values in the calling component, making that one more complicated than it needs to be.

Reply:
alright, going to use an array of errors then 🙂 just saw that this is also consistent with the argument `onUpload` callback